Cargo.toml

```diff
@@ -38,12 +38,15 @@ debug = true
 
 [dependencies]
 anyhow = "1.0"
+base64 = "0.21.0"
```

Comment:
Please truncate these to `0.21` etc., so Dependabot doesn't feel the need to bump the Z version.

Reply:
Ok, done for the others, too